zookeeper/core/cli_test.py

```diff
@@ -16,9 +16,19 @@ def run(self):
 runner = testing.CliRunner(mix_stderr=False)
 
 
+def test_command_aliases():
+    # We should be able to invoke commands with strings that resolve to the same
+    # camel-case.
+    assert runner.invoke(cli, ["test_task", "a=5"]).exit_code == 0
+    assert runner.invoke(cli, ["TestTask", "a=5"]).exit_code == 0
+    assert runner.invoke(cli, ["Test_Task", "a=5"]).exit_code == 0
+    assert runner.invoke(cli, ["testTask", "a=5"]).exit_code == 0
+    assert runner.invoke(cli, ["NotTestTask", "a=5"]).exit_code != 0
```

Comment:
I'd be fine with only supporting the proper camel case naming, but I think this is nice for usability. I can never remember if a CLI accepts snake case, camel case or something else...